lib/gui/app.js

```diff
@@ -164,7 +167,7 @@ app.run(($window, WarningModalService, ErrorService, FlashStateModel, OSDialogSe
     // Don't open any more popups
     popupExists = true;
 
-    return OSDialogService.showWarning({
+    OSDialogService.showWarning({
       confirmationLabel: 'Yes, quit',
       rejectionLabel: 'Cancel',
```

Comment:
Now that `showWarning` applies default button labels, this line could theoretically be removed ;-)
(I'm not suggesting we do though, because it probably makes the code more unreadable)